### PR TITLE
floating header extension 3

### PR DESCRIPTION
**Pull request by WesleyDRobinson**
**Description:**

Supports Floating Heade

### CODE REVIEW

File: src/components/Select/Select.module.scss [hunk 1]
```diff
@@ -13,12 +13,8 @@
     }
 
     .StyledReactSelect__multi-value {
-      margin: 0;
       padding: 2px;
-      margin-left: 0;
-      margin-bottom: 0;
-      margin-top: 2px;
-      margin-right: var(--Space-6);
+      margin: 2px var(--Space-6) 0 0;
```
Comment (WesleyDRobinson) [inline src/components/Select/Select.module.scss:17]: 🏌️

File: src/components/TextMaskedInput/TextMaskedInput.js [hunk 2]
```diff
@@ -95,45 +95,25 @@ export const TextMaskedInput = (props) => {
...
+    onBlur,
+    name,
+    placeholder: restProps.placeholder,
+    className: getClasses(),
+    disabled: restProps.disabled,
+    placeholderChar,
+    autoComplete,
+  }
+
+  if (typeof mask !== 'function') {
```
Comment (WesleyDRobinson) [inline src/components/TextMaskedInput/TextMaskedInput.js:113]: It was tough to see the difference between the two branches, I hope this helps.

File: src/components/index.d.ts [hunk 3]
```diff
@@ -325,7 +325,17 @@ export declare const CloudinaryImage: {
 }
 export declare const filePath: (publicId: any) => any
 
-export { COLORS } from './Colors'
+export declare const COLORS: {
```
Comment (WesleyDRobinson) [inline src/components/index.d.ts:328]: my IDE was not finding COLORS with the previous export 🤔 idk why, but this helps.

Comment (coleEthos) [reply src/components/index.d.ts:328]: Is fixing the IDE the only reason you made this change @WesleyDRobinson? 
My concern is anyone updating `Colors` in the future will have to know to manually change the definition here as well. That's probably okay if we're not building in new colors that often but maybe we should add a comment to `Colors.js` explaining the declaration here.

File: src/components/TextMaskedInput/TextMaskedInput.js (hunk 2 above)
Comment (coleEthos) [reply src/components/TextMaskedInput/TextMaskedInput.js:113]: Yeah that's a lot easier to read.

**Review by coleEthos — APPROVED** [review]
LGTM - just one question about future maintenance of `Colors`, but not blocking.

**Comment by coleEthos** [comment]
Reminder to re-tag with v1.2.4 @WesleyDRobinson, once done with any changes!

File: src/components/Select/Select.js [hunk 4]
```diff
@@ -23,6 +23,7 @@ export const ReactSelectComponents = components
  */
 export const Select = ({
   className,
+  classOverrides,
```
Comment (WesleyDRobinson) [inline src/components/Select/Select.js:26]: these are meant to pass thru/ override classes for the Select component itself

File: src/components/Select/Select.js [hunk 5]
```diff
@@ -122,12 +115,34 @@ export const Select = ({
...
+  const getClasses = () => {
+    const cName = className || ''
+    const cOverride = classOverrides || ''
+    const compactClass = isCompact ? styles.Compact : ''
+    return `${cName} ${compactClass} ${styles.root} ${cOverride}`
+  }
+
   return (
-    <div className={wrapperClass} data-tid={rest['data-tid']}>
+    <div className={title ? styles.wrapper : ''} data-tid={rest['data-tid']}>
```
Comment (WesleyDRobinson) [inline src/components/Select/Select.js:126]: If it makes things more "readable" I really appreciate the pattern where you do: `const wrapper = <build the styles string>` but here i think it doesn't help too much.

File: src/components/index.d.ts (hunk 3 above)
Comment (WesleyDRobinson) [reply src/components/index.d.ts]: that's a good point. definitely strange behavior on my end, and I assumed for others also. i retried the export from Colors.js (ran prettier) and that helps so far.

**Pull request merged**